themes/prism-bruha.css

```diff
@@ -0,0 +1,122 @@
+@import url(https://fonts.googleapis.com/css?family=Space+Mono);
+/**
+ * prism.js bruha for JavaScript, CoffeeScript, CSS and HTML
+ * @author Cat Carbonell
+ */
+
+code[class*="language-"],
+pre[class*="language-"] {
+
+	color: #ccc;
+	background: none;
+	font-family: 'Space Mono', Consolas, Monaco, 'Andale Mono', 'Ubuntu Mono', monospace;
+	font-size: 1em;
+	text-align: left;
+	white-space: pre;
+	word-spacing: normal;
+	word-break: normal;
+	word-wrap: normal;
+	line-height: 1.5;
+
+	-moz-tab-size: 4;
+	-o-tab-size: 4;
+	tab-size: 4;
+	-webkit-hyphens: none;
+	-moz-hyphens: none;
+	-ms-hyphens: none;
+	hyphens: none;
+}
+/* Code blocks */
+pre[class*="language-"] {
+	padding: 1em;
+	margin: .5em 0;
+	overflow: auto;
+}
+
+:not(pre) > code[class*="language-"],
+pre[class*="language-"] {
+	background: rgba(102, 93, 105, 0.8);
```

Comment:
Is the semi-transparent background color intentional?

It's a really problem with white backgrounds because the color contrasts goes to [2.47:1](https://webaim.org/resources/contrastchecker/?fcolor=CCCCCC&bcolor=857D87). It's fine on black backgrounds with [5.3:1](https://webaim.org/resources/contrastchecker/?fcolor=CCCCCC&bcolor=524A54).

Please consider choosing a darker semi-transparent bg color or `#524a54` (= the current color on a black background).